deployer/scripts/install.sh

```diff
@@ -373,6 +373,10 @@ function generate_objects() {
   generate_kibana
   generate_curator
   generate_fluentd
+
+  for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[*].metadata.name}'); do
```

Comment:
ah cool, yeah that is clearer.  it would just be shortened to `oc deploy $dc --latest`